dev-python/pyftpdlib/pyftpdlib-1.5.6-r1.ebuild

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
I suppose you can't reproduce this? Probably another case of pytest plugin interference.

```
============================================================== FAILURES ===============================================================
________________________________________________ TestFtpAuthentication.test_anon_auth _________________________________________________

self = <pyftpdlib.test.test_functional.TestFtpAuthentication testMethod=test_anon_auth>

    def tearDown(self):
        close_client(self.client)
>       self.server.stop()

self       = <pyftpdlib.test.test_functional.TestFtpAuthentication testMethod=test_anon_auth>

pyftpdlib/test/test_functional.py:90: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
pyftpdlib/test/__init__.py:379: in stop
    assert_free_resources()
        self       = <MProcessTestFTPd name='test-ftpd' pid=97 parent=82 stopped exitcode=-SIGTERM>
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    def assert_free_resources():
        ts = threading.enumerate()
        assert len(ts) == 1, ts
        p = psutil.Process()
        children = p.children()
        if children:
            for p in children:
                p.kill()
                p.wait(1)
>           assert not children, children
E           AssertionError: [psutil.Process(pid=95, status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='11:04:36')]

children   = [psutil.Process(pid=95, status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='11:04:36')]
p          = psutil.Process(pid=95, status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='11:04:36')
ts         = [<_MainThread(MainThread, started 140141243574080)>]

pyftpdlib/test/__init__.py:261: AssertionError

Reply:
Apparently not a plugin issue after all :-(.

Reply:
After various testing, just learned that if `dev-python/pytest-localftpserver` is installed, the tests for `pyftpdlib` will fail.
Therefore I would add a `!!dev-python/pytest-localftpserver` to the test dependencies.
But for now I will wait to not disturb you and CI.

About the test failure you get (I still don't get it), it might be a known upstream issue. Could you attach the full log?

But reviewing the package history and previous bug failures issues, and tests skip list, I think it is just another failing test that we should skip, sadly.